igvc/launch/localization.launch

```diff
@@ -70,12 +70,13 @@
       <param name="imu0" value="/imu"/>
       <!-- fused parameters same as above -->
       <rosparam param="imu0_config">[false, false, false,
-                                 true,  true,  true,
```

Comment:
Is there documentation anywhere for what this array of boolean values means?  Perhaps that should be included in a comment next to this value

Reply:
Good thought, but yes. The comment above this says to look above. It takes the same format as the one directly above, so the description is omitted for brevity.

Reply:
I see it now, great